owl/EASE-OBJ.owl

```diff
@@ -1895,6 +1903,14 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#Item -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#Item">
```

Comment:
not sure if Item needs to stay, is it more specific then *Patient* ?

Reply:
I think it may have been intended as a more specific Patient, but it would need refactoring in any case since "Item" is too generic a name.

The "Item" comes from some FrameNET frames, where the thing being acted on is not any Patient (which can be animate or a person), but specifically an inanimate object.